active/progress/ProgressRing.md

```diff
@@ -146,25 +45,47 @@ isn't the type's default (for example an int-typed property that doesn't default
 
 <!-- Option 2: Put these descriptions in the below API Details section,
 with a "///" comment above the member or type. -->
-Below are properties being added to ProgressRing. The IsIndeterminate property of ProgressRing will default to True for back compatability.
-
-|Name | Type | Description | 
-|:--|:-:|:-:|
-| IsIndeterminate | boolean | Defaults to True. Gets or sets a value that indicates whether the progress ring reports generic progress with a repeating pattern (indeterminate progress) or reports progress based on the Value property (determinate progress) |
-| Value | double | Gets or sets the current setting of the range control, which may be coerced. | 
-| ShowPaused | boolean | Defaults to False. Gets or sets a value that indicates whether the progress bar should use visual states that communicate a Paused state to the user.|
-| ShowError | boolean | Defaults to False. Gets or sets a value that indicates whether the progress bar should use visual states that communicate an Error state to the user. |
-| Maximum | double | Defaults to 100. Gets or sets the highest possible Value of the range element. |
-| Minimum | double | Defaults to 0. Gets or sets the Minimum possible Value of the range element. |
-| ValueChanged | event | Event that occurs when the range value changes. |
+ProgressRing can now be used in scenarios where progress is determinate or indeterminates. With this additional capability, the determinate ProgressRing does not represent a "hung" state where the user cannot interact with the app. Previously, guidance recommended that ProgressRing only be used when the user cannot continue to interact with the app, but this is no longer the only use case and ProgressRing can be used in scenarios where user interaction can continue while the ring is spinning.
```

Comment:
ProgressRing can now be used in scenarios where progress is determinate or **indeterminates**.  

suggest: indeterminate

Reply:
Should we also specifically mention scenarios such as asynchronous loading? Where user interaction can continue while the ring is spinning.

Reply:
Great idea! I will update the examples section clarifying that so it can be added to the public facing documentation. I think the API notes normally do not appear in the final doc.

Reply:
Actually I think that guidance is more appropriate in the guidelines doc for both progress controls but I agree it's important to make a note of it here as well.